### PR TITLE
[14] Connect react router

### PR DESCRIPTION
**Pull request by denyslozynskyi**
Close: #14 

Created pages folder, where present simple files for every page. Also created simple Header and Footer

### CODE REVIEW

**Review by MarynaPyvovar — APPROVED** [review]

File: src/App.tsx
[the remark this answers is not in the document]
Comment (olha-dovbush) [reply src/App.tsx:1]: We should do lazy-loading for pages

File: src/pages/DormsPage/DormsPage.tsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+function DormsPage() {
```
Comment (olha-dovbush) [inline src/pages/DormsPage/DormsPage.tsx]: It is better to use HousesPage, according to book

File: src/App.tsx [hunk 2]
```diff
@@ -1,7 +1,21 @@
...
+import HomePage from './pages/HomePage/HomePage';
+import InvestigationPage from './pages/InvestigatePage/InvestigationPage';
+import SupportPage from './pages/SupportPage/SupportPage';
+import Header from './UI/Header';
+
 function App() {
   return (
     <div>
-      <p>The Magic Place</p>
+      <Header />
```
Comment (olha-dovbush) [inline src/App.tsx:15]: Header should be in every page, so you can add it to Layout component, using react-router-dom

**Comment by olha-dovbush** [comment]
Consider all suggestions

File: src/App.tsx [hunk 3]
```diff
@@ -1,7 +1,29 @@
...
+import SettingsPage from './pages/SettingsPage/SettingsPage';
+import SpellsPage from './pages/SpellsPage/SpellsPage';
+import Footer from './UI/Footer';
+import Header from './UI/Header';
+
 function App() {
   return (
     <div>
-      <p>The Magic Place</p>
+      <Header />
```
Comment (dovbush-olha) [inline src/App.tsx:15]: Header should be in all pages, so you can move it to Outlet component from react-router-dom

File: src/App.tsx
[the remark this answers is not in the document]
Comment (denyslozynskyi) [reply src/App.tsx:1]: Ready

File: src/App.tsx (hunk 3 above)
Comment (denyslozynskyi) [reply src/App.tsx:17]: Lazy loading added

**Pull request merged**